components/squad/squad_row.lua

```diff
@@ -49,6 +52,12 @@ function SquadRow:id(args)
 	end
 
 	self.content:node(cell)
+
+	self.lpdbData['id'] = args[1]
+	self.lpdbData['nationality'] = Flags.CountryName(args[1])
```

Comment:
this seems wrong, i think it should be `args.flag` not `args[1]`

Reply:
You're right.